components/v2/Modal/Modal.tsx

```diff
@@ -2,10 +2,20 @@ import React from 'react';
 import { ModalPropsI } from './ModalProps';
 import styles from './Modal.module.scss';
 
-export const Modal: React.FC<ModalPropsI> = ({ children }) => {
+export const Modal: React.FC<ModalPropsI> = ({
+  children,
+  isClosable,
+  closeModal,
```

Comment:
Suggest to use the closeModal function as a flag for the "closability" of the modal and remove an unnecessary prop. Is it possible that isClosable === true but no closeModal func?

Reply:
yes it is possible. it is better to remove closeModal - because there is no such function